.vsts.pipelines/builds/matrix.yml

```diff
@@ -14,6 +14,7 @@ jobs:
     job: centos71
     imageName: microsoft/dotnet-buildtools-prereqs:centos-7-b46d863-20180719033416
     reportPrebuiltLeaks: true
+    generatePrebuiltBurndown: true
```

Comment:
I think this should be added to the `Production` matrix (`Production: { generatePrebuiltBurndown: true }`) rather than as a template that applies to all 4 CentOS legs. That way, any extra time will be added to the shortest leg.